hackathons.geojson

```diff
@@ -71,11 +71,33 @@
             },
             "type": "Feature"
         },
+
+
+
+	{
+	    "geometry": {
+		"coordinates": [
+		    13.33,
+		    52.44
+		],
+		"type": "Point"
+	    },
+	    "properties": {
+		"location": "Old Smithy's Dizzle - Mainzer Str. 16, 10247 Berlin, Germany",
+		"title": "POST /bank/hackathon/berlin",
+		"url": "https://www.eventbrite.de/e/post-bankhackathonberlin-registrierung-24771928505",
+		"when": "2016-09-30 - 2016-10-02"
```

Comment:
Your event should be at the end of the file since it is the last one. Events are ordered by date in the file so I do not loose overview.